lib/internal/bootstrap/context.js

```diff
@@ -19,26 +19,18 @@ const ReflectApply = global.Reflect.apply;
 const warning = 'Atomics.wake will be removed in a future version, ' +
   'use Atomics.notify instead.';
 
-let wakeWarned = false;
 function wake(typedArray, index, count) {
-  if (!wakeWarned) {
-    wakeWarned = true;
-
-    if (global.process !== undefined) {
```

Comment:
Are the changes here necessary? This will just go into the else branch when global.process is not available. Also it’s irrelevant when process.emitWarning is created because we are not going to run into this during bootstrap.

Reply:
Sure, I will discuss with @devsnek further a better approach here.